roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -5,7 +5,7 @@ apiLevels:
 - v1
 apiVersion: v1
 assetConfig:
-  logoutURL: ""
+  logoutURL: {{ openshift.master.logout_url }}
```

Comment:
This will need the default filter (if not adding a default within openshift_facts), otherwise I would expect this to throw an exception if unset.:
```
  logoutURL: {{ openshift.master.logout_url | default(None) }}
```